src/pages/solve-overview/template.html

```diff
@@ -25,7 +25,8 @@ <h3 v-html="topic.desc"></h3>
             <h1 class="rsTl-feature-title"><a class="rsTl-feature-titleLink track-cta" :href="featured.first.alias">{{ featured.first.header }}</a></h1>
             <p class="rsTl-feature-p" v-html="$options.filters.truncate(featured.first.body, 200)"></p>
             <p class="rsTl-feature-p">
-              <span class="rsTl-feature-author">{{ featured.first.field_author }}</span> – {{ getDateString(featured.first.field_published_date) }}
+              <span class="rsTl-feature-author" v-if="featured.first.field_syndicated_content === 'True'">{{ featured.first.field_attribution_name }}</span>
+              <span class="rsTl-feature-author" v-else>{{ featured.first.field_author }}</span> – {{ getDateString(featured.first.field_published_date) }}
```

Comment:
As mentioned on the www PR. We need to add the company name here too per the ticket. 

> If the syndicated content checkbox is unchecked, display the company name next to the author's name
> - Use the Company field on Author, Leadership, and Event Speaker content types
> - For example Matt Stoyka / Rackspace

I believe we'll need to add company name to the api export to accomplish this.

Reply:
This has been resolved on the Drupal side.  We're good here.